src/wallet/wallet.cpp

```diff
@@ -2344,10 +2344,10 @@ CAmount CWallet::GetAnonymizedBalance() const
         if (setWalletTxesCounted.find(outpoint.hash) != setWalletTxesCounted.end()) continue;
         setWalletTxesCounted.insert(outpoint.hash);
 
-        for (std::map<uint256, CWalletTx>::const_iterator it = mapWallet.find(outpoint.hash); it != mapWallet.end() && it->first == outpoint.hash; ++it) {
```

Comment:
Do you remember what this tried to achieve? It looks like it was initially meant to run though a map with `COutpoint` as key.

Reply:
Just some weird legacy code from times coins were indexed in another way afaik.